sound/soc/sof/intel/hda-loader.c

```diff
@@ -92,6 +92,9 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, int stream_tag)
 	int ret;
 	int i;
 
+	/* step 0: power down all host managed cores */
+	snd_sof_dsp_core_power_down(sdev, chip->host_managed_cores_mask);
+
```

Comment:
I knwo this is hamrless but is this really needed? the enabled_cores_mask will always be 0 no? we set it to 0 during suspend in sof_suspend() as well.

Reply:
@ranj063, ah, I see. I'll drop this as it is just a NOP when the modules are inserted.

btw: do we power off the cores by force when the modules are loaded or are we assume that they are off?

Reply:
they are powered off when the module is unloaded or during shutdown.